.github/workflows/push-release.yml

```diff
@@ -165,6 +165,13 @@ jobs:
           sudo rm -rf /usr/local/lib/android
       - name: Checkout code
         uses: actions/checkout@v4
+      - name: Setup Environment Variables
+        run: |
+          echo "ARCH=${{ matrix.arch }}" >> "$GITHUB_ENV"
+          source scripts/export-config
+          echo "CATTLE_RANCHER_WEBHOOK_VERSION=$CATTLE_RANCHER_WEBHOOK_VERSION" >> "$GITHUB_ENV"
```

Comment:
This isn't valid for `release/v2.7`, instead, the webhook version is manually set on the `Dockerfile.agent`